src/api/todos.ts

```diff
@@ -0,0 +1,25 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = (userId: number) => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
+};
+
+export const postTodo = (title: string, userId: number) => {
+  return client.post<Todo>('/todos', {
+    title,
+    userId,
+    completed: false,
+  });
+};
+
+export const deleteTodo = (todoId: number) => {
```

Comment:
Consider adding a comment above the `deleteTodo` function to describe its purpose 📚